### ISSUE TITLE
A C.5 Figure 16 adversarial train/test parameters

### ISSUE THREAD

**Issue by choltz95** [zjfheart/Geometry-aware-Instance-reweighted-Adversarial-Training#3]
Hi,

Great work & thanks for providing code! I was wondering about the training parameters that were used to produce the tables in Fig 16. - in particular the small CNN clean & adversarial error plots for CIFAR10.

I assumed it was l-inf, eps=0.031, but I guess this epsilon is a bit high for such a small network - I'm recovering 0.9 clean & adversarial test error during the first ~10 epochs training when I run with these parameters.

`python3 GAIRAT.py --epsilon 0.031 --net 'smallcnn' --Lambda 'inf'` 

Did I miss something in the paper re the parameters or am I running the code incorrectly?

**Comment by ZFancy** [comment]
> Hi,
> 
> Great work & thanks for providing code! I was wondering about the training parameters that were used to produce the tables in Fig 16. - in particular the small CNN clean & adversarial error plots for CIFAR10.
> 
> I assumed it was l-inf, eps=0.031, but I guess this epsilon is a bit high for such a small network - I'm recovering 0.9 clean & adversarial test error during the first ~10 epochs training when I run with these parameters.
> 
> `python3 GAIRAT.py --epsilon 0.031 --net 'smallcnn' --Lambda 'inf'`
> 
> Did I miss something in the paper re the parameters or am I running the code incorrectly?


Hi Chester,

Thank you for your attention to our work!

For training on CIFAR10 using the ```small CNN```, you can try to recover our results in Fig.16 by setting the initial learning rate to **0.01** (i.e., ```--lr-max 0.01```).

I am sorry that I omitted the specific parameters for using the small CNN. Except for the learning rate, other parameters keep the same as training with ResNet-18 (can be found at the bottom of page 20 in our paper).

Thank you very much for raising this issue! Later we will update the parameters (mainly about the learning rate) for training using ``` different networks/different datasets ``` in the README file. 

Best Regards,
Jianing

**Comment by choltz95** [comment]
Great - thanks so much!
